chain/epoch_manager/src/lib.rs

```diff
@@ -2934,4 +2930,36 @@ mod tests {
             PROTOCOL_VERSION
         );
     }
+
+    #[test]
+    fn test_protocol_version_switch_with_many_seats() {
+        let store = create_test_store();
+        let config = epoch_config(10, 1, 10, 0, 90, 60, 0);
```

Comment:
let's adjust this test to 4 block producer seats and 10 total chunk producer seats